README.md

```diff
@@ -7,6 +7,7 @@ This repository was initially bootstrapped using [CoreOS operator](https://githu
 ## Build
 
 ### Pre-requisites
+- [yamllint] version stable (https://yamllint.readthedocs.io/en/stable/quickstart.html)
```

Comment:
@tinakurian for a correct rendering:
`- [yamllint](https://yamllint.readthedocs.io/en/stable/quickstart.html) version stable`

Reply:
done!